### ISSUE TITLE
Reserve Rooms

### ISSUE THREAD

**Issue by jghibiki** [acm-ndsu/CollabCompetition#9]
ref #2

**Comment by kmerfeld** [comment]
Do we need any rooms other than qbb 104?

**Comment by meyerpa** [comment]
What's the maximum capacity of qbb 104?
Things to keep in mind:
1. If people are hanging around the whole time they may want somewhere to nap?
2. It was talked about companies can set up booths. Will this space allow booths?
3. Opening and closing ceremonies may need a raised area so people can see the presenter.

**Comment by matt-fevold** [comment]
Room 104 is an auditorium with <100 capacity so it probably not a great choice for the only room (based on it's layout/not being a good place to work on a computer). I'd recommend trying to get 102, 104, and 106. We can also utilize our ACM room 162 and the TA room that is right next door. (being a weekend we shouldn't have an issue with reserving these 3 rooms).

1. Quite possibly, there are some couches and chairs that people could potentially crash on in the QBB building. Otherwise we might want to designate a room to sleep in.

2. it depends on how many businesses want to set up booths.  
3. being an auditorium 104 should be fine for this.

**Comment by kmerfeld** [comment]
I have requested 102 104 and 106. I will update on confirmations

**Comment by jghibiki** [comment]
Companies should be able to set up booths in the atrium of the QBB as well.

**Comment by kmerfeld** [comment]
Will we need to make any special request for that?

**Comment by jghibiki** [comment]
I don't think so, as long as no one complains we are fine, but i don't think anyone will there has been a DnD group that meets in the atrium Saturdays most of the day and it makes plenty of noise so i don't foresee it being a problem.
